xbmc/video/dialogs/GUIDialogTeletext.cpp

```diff
@@ -79,6 +79,29 @@ bool CGUIDialogTeletext::OnMessage(CGUIMessage& message)
   return CGUIDialog::OnMessage(message);
 }
 
+void CGUIDialogTeletext::DoProcess(unsigned int currentTime, CDirtyRegionList &dirtyregions)
+{
+  if (m_pTxtTexture)
+  {
+    m_TextDecoder.RenderPage();
```

Comment:
I presumed that RenderPage does only update some internals and the texture which is kind of processing. The actual texture rendering takes places in Render. If that is wrong it can be corrected.